core/trino-main/src/test/java/io/trino/operator/output/TestPositionsAppender.java

```diff
@@ -241,14 +253,34 @@ public void testSliceRle()
         }
     }
 
+    @Test
```

Comment:
rename test name, if it's not byte array, then what it is?

Why the test is important (please add short comment)